### ISSUE TITLE
Translations not working for Filters

### ISSUE THREAD

**Issue by trk** [luyadev/luya-module-admin#169]
Translation key appear on filter select box.

<img width="428" alt="ekran resmi 2018-07-03 03 49 54" src="https://user-images.githubusercontent.com/455983/42192862-2e9222b8-7e74-11e8-8059-4a78565e75f8.png">

**Comment by nadar** [comment]
Yes indeed, i can confirm this issues because the name of the filter is stored while importing:

```php
$filterModel->setAttribute('name', $this->name());
```

It should be possible to display the name of the filter while output rendering.
